### DISCUSSION TITLE
공통 컴포넌트 타입 정의 위치

### DISCUSSION THREAD

**Discussion by qkdl60** [prgrms-fe-devcourse/FEDC4_CUCUMIS_Pocojang#39] [General]
기존의 타입구조는 api , model, router경로 대로 폴더가 정의 되어있는데 ,  공통 컴포넌트에서 반복적(2번 이상)으로 사용되는 타입들도 
정의가 필요한거 같아서 types> components 폴더도 필요하다고 생각됩니다.   types 안에 components 폴더도 만들어도 될까요?

**Comment by kutta97** [comment]
예전에 원지님이 BaseInput 컴포넌트에서 동일한 prop type 을 사용해서, 해당 타입의 위치에 대해서 여쭤보셨을떄,
그때, 그런 공용 컴포넌트 Prop type 들은 types/components 폴더 안에 정의하기로 얘기 나누었던 것 같습니다~!
